tests/test_livyclientfactory.py

```diff
@@ -13,8 +14,9 @@ def test_build_session_with_defaults():
     connection_string = get_connection_string("url", "user", "pass")
     kind = Constants.session_kind_pyspark
     properties = {"kind": kind}
+    ipython_display = IpythonDisplay()
```

Comment:
Can you please make `ipython_display` a `MagicMock()` and verify that its methods are called correctly? Remove the `import` after this.